reference/sectionB_elements.inc

```diff
@@ -377,44 +377,42 @@ A :code:`math` element information item (referred to in this specification as a
+|                                  | :code:`otherwise`                                              |
++----------------------------------+----------------------------------------------------------------+
+| Relational and Logical Operators | :code:`eq`, :code:`neq`, :code:`gt`, :code:`lt`, :code:`geq`,  |
+|                                  | :code:`leq`, :code:`and`, :code:`or`, :code:`xor`, :code:`not` |
++----------------------------------+----------------------------------------------------------------+
+| Arithmetic Operators             | | :code:`plus`, :code:`minus`, :code:`times`, :code:`divide`,  |
+|                                  | | :code:`power`, :code:`root`, :code:`abs`,                    |
+|                                  | | :code:`exp`, :code:`ln`, :code:`log`,                        |
+|                                  | | :code:`floor`, :code:`ceiling`,                              |
+|                                  | | :code:`min`, :code:`max`, :code:`rem`                        |
++----------------------------------+----------------------------------------------------------------+
+| Calculus Elements                | :code:`diff`                                                   |
++----------------------------------+----------------------------------------------------------------+
+| Qualifier Elements               | :code:`bvar`, :code:`logbase`, :code:`degree` \*               |
++----------------------------------+----------------------------------------------------------------+
+| Trigonometric Operators          | | :code:`sin`, :code:`cos`, :code:`tan`, :code:`sec`,          |
+|                                  |   :code:`csc`, :code:`cot`,                                    |
+|                                  | | :code:`sinh`, :code:`cosh`, :code:`tanh`, :code:`sech`,      |
+|                                  |   :code:`csch`, :code:`coth`,                                  |
+|                                  | | :code:`arcsin`, :code:`arccos`, :code:`arctan`,              |
+|                                  | | :code:`arcsec`, :code:`arccsc`, :code:`arccot`,              |
+|                                  | | :code:`arcsinh`, :code:`arccosh`, :code:`arctanh`,           |
+|                                  | | :code:`arcsech`, :code:`arccsch`, :code:`arccoth`            |
++----------------------------------+----------------------------------------------------------------+
+| Mathematical and Logical         | :code:`pi`, :code:`exponentiale`,                              |
+| Constants                        | :code:`notanumber`, :code:`infinity`,                          |
+|                                  | :code:`true`, :code:`false`                                    |
++----------------------------------+----------------------------------------------------------------+
+
+\* :code:`degree` must be a child of :code:`root` or :code:`diff`
```

Comment:
I imagine that the lack of full stop is intentional? (Just want to make sure that it isn't an oversight.)

Reply:
Just following what was there ... but as it's not a sentence, I'm fine without one